src/EditorFeatures/Core/DocumentationComments/CopilotGenerateDocumentationCommentProvider.cs

```diff
@@ -239,6 +240,8 @@ private static async Task<IReadOnlyList<ProposedEdit>> GetProposedEditsAsync(
 
             static string AddNewLinesToCopilotText(string copilotText, string? indentText, int characterLimit)
             {
+                // Double check that the resultant from Copilot does not produce any strings containing new line characters.
+                copilotText = Regex.Replace(copilotText, @"\r?\n", " ");
```

Comment:
This is just a secondary check that the model does not return text that contains any new line trivia since, if it does, it gets inserted incorrectly.